spec/graphql/language/parser_spec.rb

```diff
@@ -1,562 +1,7 @@
 require "spec_helper"
+require 'graphql/Language/parser_tests'
```

Comment:
I think this should be a little `l`, not a big `L`! Maybe it passes on OSX where the filesystem is not case sensitive?